examples/jokes/remix.config.js

```diff
@@ -6,6 +6,5 @@ module.exports = {
   assetsBuildDirectory: "public/build",
   publicPath: "/build/",
   serverBuildDirectory: "build",
-  devServerPort: 8003,
```

Comment:
@kentcdodds Not sure if this was intentional on port `8003` or not.
Can revert if wanted.

Reply:
I doubt it was intentional.